### DISCUSSION TITLE
Trying to point my domain to my reflex app

### DISCUSSION THREAD

**Discussion by NanoscaleSimulations** [reflex-dev/reflex#1707] [Q&A]
Hello there,

I am trying to point my domain to my reflex app which i have hosted on a virtual machine on oracle oci.
I can access the website by entering the ip address, but when I enter the domain, it will not connect to the website.
I have added oracle nameservers to my domain and have added A records to point to the IP address of the virtual machine.

I am starting to think that this my have something to do with the fact that I am hosting the website using the dockerfile from the reflex docker-example. I am hosting the app on port 80.

Anyone who can help me?

**Comment by masenf** [comment]
If the site is working perfectly via IP and not via domain, then it probably is an issue with the DNS. When you do something like `nslookup mydomain.com` do you see the same IP that you're able to connect to directly?

Are you exposing the reflex app via `--frontend-port 80`? or by docker port mapping? or are you running through some reverse proxy?

Some general tips, there is a frontend and a backend that both need to be exposed, either on separate ports, or by using a load balancer / reverse proxy. It's also very important to ensure the `API_URL` is set to the publicly accessible hostname+port where the backend is running (`--backend-port`).
